plugin/train.py

```diff
@@ -0,0 +1,136 @@
+import time
+
+import numpy as np
+import mxnet as mx
+
+from mxnet import gluon, nd
+from mxnet import autograd as ag
+from mxnet.gluon.data.vision import transforms
+
+from gluoncv.model_zoo import get_model
+
+
+def file_to_str(file_path):
+    with open(file_path, 'r') as file_buffer:
+        return file_buffer.read()
+
+
+def make_transform():
+    transform = transforms.Compose([
+        transforms.ToTensor(),
+    ])
+
+    return transform
+
+
+def to_mx(chip):
+    chip = nd.array(chip)
+    return chip
+
+
+def model_build(classes, pretrained_model_options=None):
+    num_cpus = 1
+    ctx = [mx.cpu(i) for i in range(num_cpus)]
+    # GluonCV Model build here
+    net = get_model('ResNet50_v2', classes=classes, ctx=ctx)
```

Comment:
There should be an option for using a pretrained model or not, but as we discussed we're not worrying about adding various options at this time. But as a default, we should be using a pretrained model since that almost always improves performance by a lot. So you should use the pretrained=True flag here. https://gluon-cv.mxnet.io/api/model_zoo.html#gluoncv.model_zoo.get_model

Reply:
The pretrained `ResNet50v2` expects `classes=1000`, whereas in the Vegas buildings example, `classes=2`. This results in an incompatible shape.

Reply:
Fixed. You can set pretrained to `True`, then change the structure of the output layer to match the number of classes.